TestUtils.js

```diff
@@ -13,6 +13,7 @@ import type {GlobalConfig, ProjectConfig} from 'types/Config';
 
 const DEFAULT_GLOBAL_CONFIG: GlobalConfig = {
   bail: 0,
+  changedFiles: null,
```

Comment:
I agree it's practical in order to propagate, but this should not be part of the config (global or project). The value has to be manually passed through

Reply:
Deleted